.buildkite/pipeline.package.mbp.yml

```diff
@@ -1,12 +1,56 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
 name: "fleet server package mbp"
```

Comment:
This name is overwriting the name or description in the catalog-info.yaml file ?